code/modules/hydroponics/beekeeping/beehive.dm

```diff
@@ -1,308 +1,455 @@
-			bee_count /= 2
-			B.fill()
-		update_icon()
-		return
+		return apply_starter_pack(I, user)
+
 	else if(istype(I, /obj/item/scanner/plant))
-		to_chat(user, "<span class='notice'>Scan result of \the [src]...</span>")
+		to_chat(user, SPAN_NOTICE("Scan result of \the [src]..."))
 		to_chat(user, "Beehive is [bee_count ? "[round(bee_count)]% full" : "empty"].[bee_count > 90 ? " Colony is ready to split." : ""]")
-		if(frames)
-			to_chat(user, "[frames] frames installed, [round(honeycombs / 100)] filled.")
-			if(honeycombs < frames * 100)
+		if(LAZYLEN(frames))
+			to_chat(user, "[LAZYLEN(frames)] frames installed, [round(honeycombs / 100)] filled.")
+			if(honeycombs < LAZYLEN(frames) * 100)
 				to_chat(user, "Next frame is [round(honeycombs % 100)]% full.")
 		else
 			to_chat(user, "No frames installed.")
-		if(smoked)
+		if(time_end_smoked >= REALTIMEOFDAY)
 			to_chat(user, "The hive is smoked.")
-		return 1
-	else if(IS_SCREWDRIVER(I))
-		if(bee_count)
-			to_chat(user, "<span class='notice'>You can't dismantle \the [src] with these bees inside.</span>")
-			return
-		to_chat(user, "<span class='notice'>You start dismantling \the [src]...</span>")
-		playsound(loc, 'sound/items/Screwdriver.ogg', 50, 1)
-		if(do_after(user, 30, src))
-			user.visible_message("<span class='notice'>\The [user] dismantles \the [src].</span>", "<span class='notice'>You dismantle \the [src].</span>")
-			new /obj/item/beehive_assembly(loc)
-			qdel(src)
-		return
+		return TRUE
 
-/obj/machinery/beehive/physical_attack_hand(var/mob/user)
-	if(!closed)
-		. = TRUE
-		if(honeycombs < 100)
-			to_chat(user, "<span class='notice'>There are no filled honeycombs.</span>")
-			return
-		if(!smoked && bee_count)
-			to_chat(user, "<span class='notice'>The bees won't let you take the honeycombs out like this, smoke them first.</span>")
+	else if(istype(I, /obj/item/grab))
+		var/obj/item/grab/G = I
+		var/mob/living/GM   = G.get_affecting_mob()
+		if(!GM)
 			return
-		user.visible_message("<span class='notice'>\The [user] starts taking the honeycombs out of \the [src].</span>", "<span class='notice'>You start taking the honeycombs out of \the [src]...</span>")
-		while(honeycombs >= 100 && do_after(user, 30, src))
-			new /obj/item/honey_frame/filled(loc)
-			honeycombs -= 100
-			--frames
-		update_icon()
-		if(honeycombs < 100)
-			to_chat(user, "<span class='notice'>You take all filled honeycombs out.</span>")
+		if(open && !notthebees && (bee_count >= 60) && (REALTIMEOFDAY > time_end_smoked))
+			user.setClickCooldown(DEFAULT_ATTACK_COOLDOWN)
+			notthebees = GM
+			user.visible_message(SPAN_DANGER("\The [user] starts jamming \the [GM]'s face into \the [src]!"), SPAN_DANGER("You start jamming \the [GM]'s face into \the [src]!"))
+			add_fingerprint(user)
+			GM.say("What-is-that? What-is-that? WHAT-IS-IT?!!")
```

Comment:
I appreciate the meme, but this is not appropriate for Neb. :P

Reply:
Aw, I mean its so obscure its more like a easter egg XD
But I can remove it. If you reaally want me to.

Reply:
Forcing people to say memes would barely fly on tg and we're nominally a roleplay fork. Please cut it.

Reply:
Done!